pkgs/applications/graphics/akira/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub
+, desktop-file-utils, meson, ninja, pkgconfig, vala_0_40, wrapGAppsHook
+, appstream-glib, cairo, glib, goocanvas2, gtk3, gtksourceview3
+, json-glib, libarchive, libgee, libxml2, pantheon, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "akira";
```

Comment:
```suggestion
  pname = "akira-unstable";
```

Reply:
pname is used by repology to identify a given package. "akira-unstable" would be a different package entirely

Reply:
https://repology.org/project/akira/versions

Reply:
then again, it looks like none of the package repositories carry it